karton/core/main.py

```diff
@@ -1,11 +1,125 @@
 import argparse
+import logging
+import os.path
+from configparser import ConfigParser
 from typing import Any, Dict, List
 
+from minio import Minio
+from redis import StrictRedis
+
 from .__version__ import __version__
 from .backend import KartonBackend
 from .config import Config
 from .karton import Consumer
 
+log = logging.getLogger(__name__)
+
+
+def get_user_option(prompt: str, default: str) -> str:
+    user_input = input(prompt + f"\n[{default}] ")
+    print("")  # just for style
+    return user_input.strip() or default
+
+
+def configuration_wizard(config_filename: str) -> None:
+    config = ConfigParser()
+
+    log.info("Configuring MinIO")
+    while True:
+        minio_access_key = get_user_option(
+            "Enter the MinIO access key", default="minioadmin"
+        )
+        minio_secret_key = get_user_option(
+            "Enter the MinIO secret key", default="minioadmin"
+        )
+        minio_address = get_user_option(
+            "Enter the MinIO address", default="localhost:9000"
+        )
+        minio_bucket = get_user_option(
+            "Enter the MinIO bucket to use", default="karton"
+        )
+        minio_secure = get_user_option('Use SSL ("0", "1")?', default="0")
+
+        log.info("Testing MinIO connection...")
+        minio = Minio(
+            endpoint=minio_address,
+            access_key=minio_access_key,
+            secret_key=minio_secret_key,
+            secure=bool(int(minio_secure)),
+        )
+        bucket_exists = False
+        try:
+            bucket_exists = minio.bucket_exists(minio_bucket)
+        except Exception as e:
+            log.info("Error while connecting to MinIO: %s", e)
```

Comment:
Maybe it would be better to show full exception? A bit dirty, but could be more helpful for debugging the purpose.
```suggestion
            log.info("Error while connecting to MinIO: %s", e, exc_info=True)
```